ssz/sedes/base.py

```diff
@@ -73,6 +76,10 @@ def get_hash_tree_root(self, value: TSerializable) -> bytes:
     def chunk_count(self) -> int:
         pass
 
+    @abstractmethod
+    def get_key(self, value: Any) -> bytes:
+        pass
```

Comment:
We recently switched to doing `abstractmethod` this way.

```python
@abstractmethod
def get_key(self, value: Any) -> bytes:
    ...
```

Thoughts?  Intended to help visually separate the `abstractmethod` empty body from an actual empty function body.